external/CMakeLists.txt

```diff
@@ -170,5 +170,13 @@ add_subdirectory(SQLiteCpp)
 
 add_subdirectory(Catch2)
 
-add_subdirectory(pybind11 EXCLUDE_FROM_ALL)
+if(NOT MINGW AND NOT ANDROID)
+  set(USE_PYBIND_DEFAULT ON)
+else()
+  set(USE_PYBIND_DEFAULT OFF)
+endif()
+option(BUILD_PYBIND "build python bindings" ${USE_PYBIND_DEFAULT})
```

Comment:
I think this option declaration/default should move back to the root CMakeLists.txt, since it also affects the `pybind/` directory.